tests/ctst/world/Zenko.ts

```diff
@@ -156,8 +143,8 @@ export default class Zenko extends World<ZenkoWorldParameters> {
         }
 
         // Workaround to be able to access global parameters in BeforeAll/AfterAll hooks
-        CacheHelper.parameters = this.parameters;
-        this.cliMode.parameters = this.parameters as ClientOptions;
+        CacheHelper.parameters = this.parameters as unknown as Record<string, unknown>;
```

Comment:
this is required until fixed in CTST (https://scality.atlassian.net/browse/CTST-34)